python_modules/dagster/dagster/_core/definitions/resource_definition.py

```diff
@@ -107,6 +107,8 @@ def __init__(
         if version:
             experimental_arg_warning("version", "ResourceDefinition.__init__")
 
+        self._dagster_maintained = False
```

Comment:
TODO - add documenting comment that this attribute is modified by the dagster_maintained decorators